src/PrestaShopBundle/Controller/Admin/TranslationsController.php

```diff
@@ -194,11 +196,19 @@ protected function findLanguageByLocale($locale)
     protected function getTranslationsCatalogue(Request $request)
     {
         $lang = $request->request->get('lang');
+        $type = $request->request->get('type');
 
         $translator = $this->container->get('translator');
 
         $locale = $this->langToLocale($lang);
 
+        $tree = $this->get('ps.translations_factory')->createTree($type, $locale);
```

Comment:
To be consistent with the remaining part of this method, I would the assign the factory return value to a `$catalogue` variable instead of `tree`. Moreover in this controller, the tree is actually prepared later on to before be passed to the Twig translation extension (See https://github.com/mickaelandrieu/PrestaShop/blob/76f83eaf98ef5f94422adcd0d02f49b8664e4079/src/PrestaShopBundle/Controller/Admin/TranslationsController.php#L62). Wdyt?